ros2_serial_example/src/ros2_to_serial_bridge.cpp

```diff
@@ -38,7 +38,7 @@ constexpr int BUFFER_SIZE = 1024;
 
 namespace ros2_to_serial_bridge
 {
-ROS2ToSerialBridge::ROS2ToSerialBridge() : rclcpp::Node("ros2_to_serial_bridge")
+ROS2ToSerialBridge::ROS2ToSerialBridge(const rclcpp::NodeOptions& node_options) : rclcpp::Node("ros2_to_serial_bridge", "", node_options)
```

Comment:
Hm, I don't think we actually need the second, empty argument when initializing `rclcpp::Node`.  That is, I think you should just be able to call:

```
rclcpp::Node("ros2_to_serial_bridge", options)
```